src/views/poi_panel.dot

```diff
@@ -110,9 +116,32 @@
         {{?}}
       </div>
 
+      {{? isAnywherePoi && this.categories }}
+          <div class="service_panel__categories--poi">
+            <hr>
+            <h3 class="service_panel__categories_title">
+              <span class="icon-icon_compass"></span>{{= _("Search around this place", "poi") }}
+            </h3>
+            {{~ this.categories:category:index }}
+              {{?
+                !this.mustDeployCategories ||
+                (this.mustDeployCategories && !this.isDeployed && index + 1 < 8) ||
+                (this.mustDeployCategories && this.isDeployed)
+              }}
+                <button class="service_panel__category" type="button" {{= click(this.openCategory, this, category) }}>
+                  <div class="service_panel__category__icon" style="background: {{= category.backgroundColor }}">
+                    <span class="icon icon-{{= category.iconName }}"></span>
+                  </div>
+                  <div class="service_panel__category__title">{{= category.label }}</div>
+                </button>
+            {{?}}
+          {{~}}
+          </div>
+        {{?}}
+
       {{= this.PoiBlocContainer.render(this.poi) }}
 
-      {{? this.poi.isFromOSM() }}
+      {{? isAnywherePoi || (this.poi.isFromOSM && this.poi.isFromOSM()) }}
```

Comment:
this block should not be visible with anywhere POIs: OSM view and edit URLs are not defined for them

Reply:
Fixed https://github.com/QwantResearch/erdapfel/pull/289/commits/a00d7b79c4c72c7cdc06a892332609492fb6a030